app/integrations/github/client.js

```diff
@@ -0,0 +1,127 @@
+const { App } = require("octokit");
+
+const getJiraIssueFromPullRequestTitle = function (pullRequest) {
+  let jiraIssueKey =  ""
+  const jiraIssueKeyRegex = /(bot|test|chore|fix|feat)\(([^)]+)\)/
+  var arr = jiraIssueKeyRegex.exec(pullRequest.title);
+  if (arr?.length === 3) {
+    jiraIssueKey = arr[2]
+  } else {
+    throw new Error(`Please, name your PR according to SSENSE [standards](https://homebase.ssense.com/docs/default/Component/development-standards/standards/git-practices/nomenclature/#commit-message-format)!`);
+  }
+  return jiraIssueKey
+}
+
+const getSectionFromText = function (s, section) {
```

Comment:
Improves clarity by making it more explicit what this variable represents.
```suggestion
const getSectionFromText = function (text, section) {
```